build.gradle

```diff
@@ -665,7 +665,13 @@ ext {
                 version: lombokVersion,
                 group  : 'org.projectlombok',
                 name   : 'lombok'
-            ]
+            ],
+            'micronaut.jms': [
+                    version:micronautJmsVersion,
+                    group:'io.micronaut.jms',
+                    name:'micronaut-jms',
```

Comment:
Is there a `micronaut-jms` module? I don't think so.

As far as I understand this, the `name` property is for modules with just one artifact, where `modules` is for registering additional artifacts.

@graemerocher correct me if I'm wrong

Reply:
Yes you are right `name` should be `micronaut-jms-core` and the core module removed from the other modules